theme/parts/tabsbar.css

```diff
@@ -594,10 +596,18 @@ tab-group {
 			margin-inline-end: 3px;
 			width: 16px;
 		}
+
 		tab-group:not([collapsed]) &::before {
 			background-image: var(--gnome-icon-pan-down-symbolic);
 		}
 	}
+
+	&[aria-label="Unnamed Group"] {
```

Comment:
Better use `[tooltiptext=""]`, otherwise it will only work when the browser is set to English.